docker_images/diffusers/app/pipelines/text_to_image.py

```diff
@@ -0,0 +1,33 @@
+import os
+from typing import TYPE_CHECKING
+
+from app.pipelines import Pipeline
+from diffusers import DiffusionPipeline
+
+
+if TYPE_CHECKING:
+    from PIL import Image
+
+
+class TextToImagePipeline(Pipeline):
+    def __init__(self, model_id: str):
+
+        kwargs = (
+            {"safety_checker": None}
+            if model_id.startswith("hf-internal-testing/")
+            else {}
+        )
+        self.ldm = DiffusionPipeline.from_pretrained(
+            model_id, use_auth_token=os.getenv("HF_API_TOKEN"), **kwargs
+        )
```

Comment:
Should we set a seed or should we have no seed?

Reply:
Why do we need a seed ? By default there's no seed and we just generate at random, and that's good IMO.